package.json

```diff
@@ -8,8 +8,7 @@
   },
   "dependencies": {
     "apollo-client": "^1.9.0",
-    "auth0-js": "^8.8.0",
-    "auth0-lock": "^10.19.0",
```

Comment:
@wdoug do we loose the auth0 lock interface ? its cool but we'll have to come up with a login interface of our own ? or maybe we add auth0 lock later?

Reply:
Talked to @wdoug RT / we'll work on a login view later or add this package as needed.